recipes/r-bas/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set version = '1.5.5' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-bas
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/BAS_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/BAS/BAS_{{ version }}.tar.gz
+  sha256: 9222694198c9ec62f0bf556881c823ab0e5e247335d5737437d45292cde853c5
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - libblas
+    - libcblas
+    - liblapack
+    - liblapacke
```

Comment:
libblas and liblapack are rquired... are libcblas and liblapacke really also required?

Reply:
No, good catch! I've cleaned it up.